index.html

```diff
@@ -89,6 +89,212 @@ <h1 class="presentation">
           </ul>
         </div>
       </section>
+      <section class="all_works">
+        <article class="works">
+          <div class="works__image">
+            <img src="images/mobile/1.png" alt="work 1" class="img1" />
+          </div>
+          <div class="work__description">
+            <h2>Tonic</h2>
+            <div class="short_desc">
+              <h4>CANOPY</h4>
+              <span></span>
+              <h4>Back end Dev</h4>
+              <span></span>
+              <p>2015</p>
+            </div>
+            <div class="long_desc">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="work__lang">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <button type="button" class="btn btn-enabled">See project</button>
```

Comment:
- [ ] Kindly make sure the margin between the cards and the sides of the page matches the original design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/158474481-37c2d737-95f9-4f78-8081-eb7362f9e592.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/158474400-953ad09c-0c54-42b0-83e2-a3d5ee58dd9e.png)